tests/conftest.py

```diff
@@ -185,6 +188,14 @@ def item_submission_instance(dspace_metadata):
     )
 
 
+@pytest.fixture
+def mocked_item_db():
+    with mock_aws():
+        if not ItemDB.exists():
+            ItemDB.create_table()
+        yield
```

Comment:
Is this really all that's needed to mock a functional DynamoDB table!?  If so, that's fantastic.

Reply:
I believe so!!!